whipper/test/common.py

```diff
@@ -68,14 +68,15 @@ def readCue(self, name):
         version so we can use it in comparisons.
         """
         ret = open(os.path.join(os.path.dirname(__file__), name)).read(
-            ).decode('utf-8')
+        ).decode('utf-8')
```

Comment:
Maybe instead do something like:
```python
        cuefile = os.path.join(os.path.dirname(__file__), name)
        ret = open(cuefile).read().decode('utf-8')
```

This still splits it up in two lines, but instead of continuing one line's logic into the next, it makes the two lines have distinct pieces of the logic, making it easier to follow along in the code.